asab/library/service.py

```diff
@@ -36,9 +45,9 @@ async def read(self, path):
 			if item is not None:
 				return item
 
-	async def list(self, path, recursive=True):
+	async def list(self, path, tenant, recursive=True):
```

Comment:
`tenant` should not be introduced here - or at least optionally, if you need that for "disable" evaluation.